editor.planx.uk/src/pages/FlowEditor/components/Sidebar/index.tsx

```diff
@@ -23,19 +26,24 @@ import StyledTab from "./StyledTab";
 
 type SidebarTabs = "PreviewBrowser" | "History" | "Search" | "Console";
 
-const Root = styled(Box)(({ theme }) => ({
+const SIDEBAR_WIDTH = 500;
+const SIDEBAR_WIDTH_MINIMISED = 20;
```

Comment:
nit: should these be set directly as strings to avoid the extra templating step below?